package.json

```diff
@@ -51,27 +64,41 @@
   "license": "MIT",
   "dependencies": {
     "async": "^2.6.0",
+    "debug": "^3.1.0",
     "detect-node": "^2.0.3",
     "eslint-config-standard-jsx": "^4.0.2",
     "go-ipfs-dep": "0.4.13",
-    "ipfs-api": "^17.1.3",
+    "hapi": "^16.6.2",
+    "hat": "0.0.3",
+    "http-browserify": "^1.7.0",
+    "ipfs": "^0.27.5",
+    "ipfs-api": "^17.2.0",
+    "lodash.clonewith": "^4.5.0",
+    "lodash.defaultsdeep": "^4.6.0",
+    "lodash.get": "^4.4.2",
+    "lodash.mapvalues": "^4.6.0",
     "multiaddr": "^3.0.1",
     "once": "^1.4.0",
+    "qs": "^6.5.1",
+    "readable-stream": "^2.3.3",
     "rimraf": "^2.6.2",
+    "safe-json-parse": "^4.0.0",
     "shutdown": "^0.3.0",
+    "stream-http": "^2.7.2",
     "subcomandante": "^1.0.5",
+    "superagent": "^3.8.2",
     "truthy": "0.0.1"
   },
   "devDependencies": {
     "aegir": "^12.2.0",
     "chai": "^4.1.2",
     "dirty-chai": "^2.0.1",
-    "ipfs": "^0.26.0",
     "is-running": "1.0.5",
     "mkdirp": "^0.5.1",
     "multihashes": "~0.4.12",
     "pre-commit": "^1.2.2",
-    "safe-buffer": "^5.1.1"
+    "safe-buffer": "^5.1.1",
```

Comment:
This is not used anymore, can be deleted.

Reply:
cleaning up deps and pushing shortly.